pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -129,9 +131,12 @@ def update(self, args):
         if self.nvim_treesitter_updated:
             print("updating nvim-treesitter grammars")
             nvim_treesitter_dir = ROOT.joinpath("nvim-treesitter")
-            lockfile = json.load(open(args.nixpkgs.join(NVIM_TREESITTER_GENERATED_FILE, "lockfile.json")))
+            lockfile = json.load(open(os.path.join(os.environ['NVIM_TREESITTER'], "lockfile.json")))
```

Comment:
the goal of the last update was to decouple the updater from nixpkgs, so you could run it on arbitrary input files so here instead I would use `self.nixpkgs_repo` to find the root of the checkout

Reply:
Here we are not looking for the path of the repo but the nvim-treesitter derivation path right ? 
I.e. `/nix/store/xxxxxxx-nvim-treesitter/lockfile.json`